tool/bundler/standard26_gems.rb.lock

```diff
@@ -78,4 +83,4 @@ DEPENDENCIES
   test-unit
 
 BUNDLED WITH
-   2.4.13
+   2.5.0.dev
```

Comment:
Do we need dev bundler here?

Reply:
It has to switch bundler version during install if it's not the same as the current version, but it's probably not important either way.

Reply:
I pushed using 2.4.20 instead.